src/webgpu/format_info.ts

```diff
@@ -1554,6 +1554,26 @@ export const kValidTextureFormatsForCopyE2T = [
   'rgba32float',
 ] as const;
 
+/** List of storage texture formats */
+export const kStorageTextureFormats = [
```

Comment:
Thanks. Changed to use `TexelFormats` and filed an issue about `bgra8unorm`. 

https://github.com/gpuweb/cts/issues/3854

I think we should address that globally since every place that uses `TexelFormats` should probably be testing `bgra8unorm`